src/components/WorkOrder/EditWorkOrder.tsx

```diff
@@ -8,17 +8,14 @@ import CharacterCountLimitedTextArea from '../Form/CharacterCountLimitedTextArea
 import Spinner from '../Spinner'
 import ErrorMessage from '../Errors/ErrorMessage'
 
-import {
-  EditWorkOrderProps,
-  FormValues,
-} from '../../types/edit-workorder/types'
+import { EditWorkOrderProps, FormValues } from '../../types/generic/types'
```

Comment:
Why not keep these types within the current file? Im not sure what advantage this has? `FormValues` is not generic, is only ever used in this file